### PR TITLE
Update AddAzureVirtualNetworkPeeringCommand.cs

### PR DESCRIPTION
**Pull request by YuKogasaka**
I think that AlloowGatewayTransit is typo.

Correct option  is AllowGatewayTransit.

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kogesaka__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Comment by shahabhijeet** [comment]
@kogesaka is this a breaking change as the name of a public API has changed?

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

**Comment by kogesaka** [comment]
@shahabhijeet Yes, in addition, I think that "vnetPeering.AllowGatewayTransit = this.AlloowGatewayTransit;" in Line 94 need to fix.

**Comment by shahabhijeet** [comment]
Asked networking team to take a look at this change, though minor would like their opinion on it.
Also can we add alias to the parameter name if we decide to go ahead with the rename to maintain parity.

**Comment by DeepakRajendranMsft** [comment]
LGTM. we have to also update the test script to fix the test failure.

File: src/ResourceManager/Network/Commands.Network/VirtualNetwork/Peering/AddAzureVirtualNetworkPeeringCommand.cs [hunk 1]
```diff
@@ -55,7 +55,7 @@ public class AddAzureVirtualNetworkPeeringCommand : VirtualNetworkPeeringBase
         [Parameter(
             Mandatory = false,
             HelpMessage = "Flag to allow gatewayLinks be used in remote virtual network's link to this virtual network")]
-        public SwitchParameter AlloowGatewayTransit { get; set; }
```
Comment (markcowl) [inline src/ResourceManager/Network/Commands.Network/VirtualNetwork/Peering/AddAzureVirtualNetworkPeeringCommand.cs]: Please add an Alias, so existing scripts will not be broken

**Comment by markcowl** [comment]
@DeepakRajendranMsft  @kogesaka Please add a parameter alias to the old proeprty name - this will also fix the tests, btw.

**Comment by markcowl** [comment]
@kogesaka @DeepakRajendranMsft We will close this pull request this week if you do not respond to review comments.  Please respond or this PR will e closed at EOW.

**Comment by DeepakRajendranMsft** [comment]
created PR : https://github.com/Azure/azure-powershell/pull/2865

**Comment by markcowl** [comment]
Clsoing in favor of #2865

**Pull request closed without merging**